noodle-app-css/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Bootstrap Noodles!</title>
+    <!-- CSS only -->
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-md navbar-dark bg-dark mb-4">
+      <div class="container-fluid" id="nav">
+        <a class="navbar-brand logo" href="#">Noodle Recipe Website</a>
+      </div>
+    </nav>
+    <main class="container">
+      <h1 class="text-center mb-4" id="noodle-app-mobile">Noodle App</h1>
+      <div class="row mb-4" id="header-image-desktop">

```

Comment:
for example instead of using `id` here, you can change to use the utility class from bootstrap
```suggestion
      <div class="row mb-4 d-none d-lg-block">

```